tests/test_completenessPlot.py

```diff
@@ -0,0 +1,183 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+import os
+import shutil
+import tempfile
+import unittest
+
+import lsst.utils.tests
+import matplotlib.pyplot as plt
+import numpy as np
+from lsst.analysis.tools.actions.plot import CompletenessHist
+from lsst.analysis.tools.actions.plot.plotUtils import get_and_remove_figure_text
+
+# matplotlib.use("Agg")
```

Comment:
Comment

Reply:
Actually this needs to be there or else pytesting the file will actually open a plot window, although weirdly that doesn't happen with scons so scons must also be setting that somewhere.

Reply:
I added a boolean (default `False`) to disable this for debugging tests.